src/phantom.js

```diff
@@ -315,8 +322,10 @@ export default class Phantom {
     }
 
     _heartBeat(): void {
-        if (this.commands.size === 0) {
-            this.execute('phantom', 'noop');
+        if (this.commands.size === 0 && !this.doing_NOOP) {
```

Comment:
ok, good to know you saw this locally. Can you change `doing_NOOP` to something more meaningful? Like `NoopInProgress`?